artemis/config.py

```diff
@@ -611,6 +611,29 @@ class DomainExpirationScanner:
                 int, "The scanner warns if the domain's expiration date falls within this time frame from now."
             ] = get_config("DOMAIN_EXPIRATION_TIMEFRAME_DAYS", default=30, cast=int)
 
+        class SqlInjectionDetector:
+            SQL_INJECTION_DETECTOR_MAX_BATCH_SIZE: Annotated[
+                int,
+                "How many sites to scan at once. This is the maximum batch size",
+            ] = get_config("SQL_INJECTION_DETECTOR_MAX_BATCH_SIZE", default=10, cast=int)
+
+            THRESHOLD: Annotated[
+                int,
+                "Seconds to method sleep() or pg_sleep()",
+            ] = get_config("THRESHOLD", default=1, cast=int)
+
+            SLEEP_PAYLOAD: Annotated[
```

Comment:
SQL_INJECTION_SLEEP_PAYLOADS, as it's list, right?

Reply:
Yes. Done.